### PR TITLE
adds config for vms used during admin exam

### PR DESCRIPTION
**Pull request by wizzdom**

### CODE REVIEW

File: jobs/user-vms/admin-exams.hcl [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+          "bridge,id=hn0",
+          "-device",
+          "virtio-net-pci,netdev=hn0,id=nic1,mac=52:54:84:ba:49:20",
+          "-smbios",
+          "type=1,serial=ds=nocloud-net;s=http://136.206.16.5:8000/ayden-vm/",
+        ]
+      }
+    }
+  }
+  group "arkues-vm" {
```
Comment (DistroByte) [inline jobs/user-vms/admin-exams.hcl:49]: I think we can generalise this a little, no need to commit every user. Perhaps commit users who have passed the exam?

Comment (wizzdom) [reply jobs/user-vms/admin-exams.hcl]: Cleaned this up, as well as adding some comments for clarification

File: jobs/user-vms/admin-exams.hcl [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+      config {
+        image_path = "local/ayden-vm.qcow2"
+        accelerator = "kvm"
+        drive_interface = "virtio"
+
+        args = [
+          "-netdev",
+          "bridge,id=hn0",
+          "-device",
+          "virtio-net-pci,netdev=hn0,id=nic1,mac=52:54:84:ba:49:20", # mac address must be unique or else you will regret it
```
Comment (DistroByte) [inline jobs/user-vms/admin-exams.hcl:42]: Add this comment to the README in `jobs/user-vms` too please, or create a new PR with an update for `gh.com/redbrick/docs`.

Comment (wizzdom) [reply jobs/user-vms/admin-exams.hcl:42]: Added the note there in the README. Will also add this in the docs repo

**Pull request merged**